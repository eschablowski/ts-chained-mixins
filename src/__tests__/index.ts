import { describe, expect, test } from "@jest/globals";

import * as indx from "..";

describe("index.ts", () => {
    test("must match from previous api", () => {
        expect(indx).toMatchSnapshot();
    });
});
