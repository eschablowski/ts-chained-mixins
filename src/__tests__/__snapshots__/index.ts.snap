// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index.ts must match from previous api 1`] = `
{
  "NotMixableError": [Function],
  "chainableMixinsMixin": [Function],
  "default": [Function],
}
`;
